### DISCUSSION TITLE
Unit test not firing back a change event

### DISCUSSION THREAD

**Discussion by globalrewardsusa-vahava** [sagalbot/vue-select#1509] [Help Wanted]
I've tried creating a unit test for v-select but with everything I do, I cannot have the component fire back a change event.

element : 

<v-select data-testid="shippingState" :options="states" :selected="form.cardState" @input="changeShippingState" />

In this example I'm using @testing-library/vue

    const shippingState = getByTestId('shippingState');  // I'm getting an instance 
    const inpt = shippingState.querySelector('input'); // and here too - HTMLInputElement
    await fireEvent.click(inpt)
    await fireEvent.update(inpt, "NJ")
    await fireEvent.keyDown(inpt);

I can see the value is set but @input="changeShippingState" callback is never called.

**Comment by trumpet2012** [comment]
@globalrewardsusa-vahava Did you ever discover a work around for this? Currently running into this exact issue and have had no luck finding a solution.
